src/Configuration/src/Kubernetes.ServiceBinding/LegacyConnectorsPostProcessors.cs

```diff
@@ -19,17 +19,19 @@ public void PostProcessConfiguration(PostProcessorConfigurationProvider provider
 
         configData.Filter(ServiceBindingConfigurationProvider.KubernetesBindingsPrefix, ServiceBindingConfigurationProvider.TypeKey, BindingTypeKey).ForEach(
             bindingNameKey =>
-            {
-                // Spring -> spring.rabbitmq....
-                // Steeltoe -> rabbitmq:client:....
-                var mapper = new ServiceBindingMapper(configData, bindingNameKey, "rabbitmq", "client");
-                mapper.MapFromTo("addresses", "uri");
-                mapper.MapFromTo("host", "server");
-                mapper.MapFromTo("password", "password");
-                mapper.MapFromTo("port", "port");
-                mapper.MapFromTo("username", "username");
-                mapper.MapFromTo("virtual-host", "virtualhost");
-            });
+        {
+            var mapper = new ServiceBindingMapper(configData, bindingNameKey, "mongodb", "client");
+
+            // Mapping from Kubernetes secrets to driver-specific connection string parameters.
+            // At the time of writing (November 2023), there's no complete official documentation for the available secrets. Some pointers:
+            // - Generic secrets: https://github.com/servicebinding/spec#well-known-secret-entries
+            // - Input keys used at https://github.com/spring-cloud/spring-cloud-bindings/blob/main/spring-cloud-bindings/src/main/java/org/springframework/cloud/bindings/boot/MongoDbBindingsPropertiesProcessor.java
```

Comment:
This is not true: it does not map to driver-specific connection string parameters, but to the properties on `MongoDbConnectorOptions`. I find this comment misleading, because it implies this was investigated/tested and decisions were made. But that's not the case, it's just an untested copy from main and it differs considerably from the non-legacy post-processor. So I believe that intentionally being vague here is the best thing to do, unless extra effort is put in to verify this actually works.

```suggestion
            // Spring -> spring.data.mongodb....
            // Steeltoe -> mongodb:client:...
```